sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/integrationtest/BeamSqlDateFunctionsIntegrationTest.java

```diff
@@ -50,7 +51,9 @@
         .addExpr("SECOND(ts)", 26L)
         .addExpr("FLOOR(ts TO YEAR)", parseDate("1986-01-01 00:00:00"))
         .addExpr("CEIL(ts TO YEAR)", parseDate("1987-01-01 00:00:00"))
-        ;
+        .addExpr("TIMESTAMPADD(DAY, 3, TIMESTAMP '1984-04-19 01:02:03')",
```

Comment:
can you add more elements to cover YAER, MONTH, DAT, HOUR, MINUTE, SECOND?

Reply:
Sure, I'll add them here. Plus I will check if the infix version works correctly